From d0e3c55d0421f6c0d6ed9e2169280aafb0c2b9c6 Mon Sep 17 00:00:00 2001
From: sebastien baudouin <sebastien.baudouin@windriver.com>
Date: Mon, 30 Mar 2015 17:01:50 +0200
Subject: [PATCH 1/4] browser: missing method for connect

This commit remove the connect on a missing method causing error at execution
time.
---
 browser/browserpage.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/browser/browserpage.h b/browser/browserpage.h
index bede6d9..94ac347 100644
--- a/browser/browserpage.h
+++ b/browser/browserpage.h
@@ -31,8 +31,8 @@ public:
                 m_userInput, SIGNAL(onConfirmDialog(QString)));
         connect(this,        SIGNAL(onPromptDialog(QString, QString)),
                 m_userInput, SIGNAL(onPromptDialog(QString, QString)));
-        connect(this,        SIGNAL(onDialogCanceled(void)),
-                m_userInput, SIGNAL(onDialogCanceled(void)));
+        //connect(this,        SIGNAL(onDialogCanceled(void)),
+        //        m_userInput, SIGNAL(onDialogCanceled(void)));
         }
     ~BrowserPage(){ }
 
-- 
1.9.1

